### DISCUSSION TITLE
slack

### DISCUSSION THREAD

**Discussion by masahiko-ikau-com** [innovation1005/linear-algebra#15] [Q&A]
ありがとうございます。ただ、招待URLをいただくか、メンバーのメールアドレスを登録いたけないとログインできないように思います。
（試して見ましたが弾かれてしまいました。）

**Comment by innovation1005** [comment]
***@***.*** ***@***.***> 

です。

 

From: masahiko-ikau-com ***@***.***> 
Sent: Thursday, July 7, 2022 9:41 PM
To: innovation1005/linear-algebra ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [innovation1005/linear-algebra] slack (Discussion #15)

 

ありがとうございます。ただ、招待URLをいただくか、メンバーのメールアドレスを登録いたけないとログインできないように思います。
（試して見ましたが弾かれてしまいました。）

**Reply by masahiko-ikau-com** [reply]
ありがとうございます。
以下であっていますでしょうか。

・いただいたslack名からスペースを除いたurlで登録されている
・すでにメールアドレスでメンバー登録いただいている
・上の※を適切に設定したものがパスワードである

すみません。上記を仮定してログインを試したのですが、ユーザー名かパスワードのエラーでログインできませんでした。来週オンラインで教えてくださると幸いです。
